src/main/java/com/spring/beans/Bicycle.java

```diff
@@ -0,0 +1,43 @@
+package com.spring.beans;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Component;
+
+@Setter
+@Getter
+@Component
+@NoArgsConstructor
+@PropertySource({"classpath:spring/bicycle.properties"})
+public class Bicycle implements Transport {
+    private final static Logger LOG = Logger.getLogger(Car.class.getName());
+
+    @Value("${speed}")
+    private int speed;
+
+    @Value("${countOfWheels}")
+    private int countOfWheels;
+
+    @Autowired
+    @Qualifier("summerWheel")
+    private Wheel wheel;
+
+    public Bicycle(Wheel wheel, int countOfWheels, int speed) {
```

Comment:
whu dont you want to use lombok annotation for its purpose?

Reply:
I resolved it